static/js/services/videowaiter.js

```diff
@@ -67,15 +89,16 @@ define(["underscore"], function(_) {
 		// videoWaiter wait
 		return {
 			wait: function(video, stream, cb, err_cb) {
-				var waiter = new Waiter();
+				var waiter = new Waiter(video, stream, cb, err_cb);
 				_.defer(function() {
-					waiter.start(video, stream, cb, err_cb);
+					waiter.start();
 				});
-				return waiter;
+				return {
```

Comment:
This change seems unrelated - why is this changed. This is also a functional change - if required please make this another commit or PR.